### PR TITLE
get rid of useless tests on s390

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: main.pm [hunk 1]
```diff
@@ -481,7 +481,7 @@ sub load_consoletests() {
             }
             loadtest "console/snapper_undochange.pm";
         }
-        if (get_var("DESKTOP") !~ /textmode/) {
+        if (get_var("DESKTOP") !~ /textmode/ && !check_var("BACKEND", "s390x")) {
```
Comment (coolo) [inline main.pm]: please check ARCH instead of BACKEND - remember zkvm

File: main.pm [hunk 2]
```diff
@@ -498,7 +498,7 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
-        if (!get_var("OFW")) {
+        if (!get_var("OFW") && !check_var("BACKEND", "s390x")) {
             loadtest "console/glibc_i686.pm";
```
Comment (coolo) [inline main.pm]: this is even worse as it should check_var('ARCH', 'x86_64') - it's pointless indepedent of the firmware or the backend used.

File: main.pm [hunk 3]
```diff
@@ -498,7 +498,7 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
-        if (!get_var("OFW")) {
+        if (!get_var("OFW") && check_var("ARCH", "x86_64")) {
```
Comment (dirkmueller) [inline main.pm]: !get_var("OFW") && 

can be removed now.

Comment (sysrich) [inline main.pm]: The OFW var has become used throughout the codebase as effectivly equivalent to "ARCH like ppc64*" - given how we use the variable, are you sure your statement is correct?

Comment (dirkmueller) [inline main.pm]: Yes, we don't want this to be run on aarch64 either. The check makes only sense on x86_64

Comment (coolo) [inline main.pm]: !OFW and x86_64 is kind of duplicated

**Comment by coolo** [comment]
fixed and commited as 9dc7fda

**Pull request closed without merging**